test/test_formatters.rb

```diff
@@ -45,8 +45,19 @@ def setup
       'this < that'               => 'this &lt; that',
       "it's complicated"          => "it\\'s complicated",
       'a "good" way'              => 'a \"good\" way',
-      '<b>bold</b>'               => '&lt;b>bold&lt;/b>',
-      '<a href="target">link</a>' => '&lt;a href=\"target\">link&lt;/a>',
+
+      '<b>bold</b>'               => '<b>bold</b>',
+      '<i>italic</i>'             => '<i>italic</i>',
+      '<u>underline</u>'          => '<u>underline</u>',
+      '<a href="target">link</a>' => '<a href=\"target\">link</a>',
```

Comment:
I think this should be:
`'<a href="target">link</a>' => '<a href="target">link</a>',`

As far as I can see the double quotes inside the href should not be escaped by a backslash.

Reply:
I verified that the quotation marks should not be escaped:

strings.xml:
```
<resources>
    <string name="app_name">My Application</string>
    <string name="action_settings">Settings</string>
    <string name="placeholder">placeholder</string>
    <string name="test_string">You can add a <a href=\"http://www.google.com\">link</a> to text too.</string>
</resources>
```

Compiler error:
```
Error:Execution failed for task ':app:mergeDebugResources'.
> /Users/jpmeijers/AndroidStudioProjects/MyApplication/app/src/main/res/values/strings.xml:5:54: Error: Open quote is expected for attribute "href" associated with an  element type  "a".
```

It works correctly when the quotation marks are not escaped:
```
<string name="test_string">You can add a <a href="http://www.google.com">link</a> to text too.</string>
```